src/app/collection/pages/nice-challenge/components/philosophy/philosophy.component.scss

```diff
@@ -0,0 +1,119 @@
+@import '~_vars.scss';
+@import '../../../../collection-globals.scss';
+
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+
+  .philosophy-content {
+    display: flex;
+    flex-basis: 75%;
+    justify-content: center;
+
+    .grid {
+      width: 100%;
+      display: grid;
+      grid-template-columns: 50% 50%;
+      grid-row: auto auto;
+      justify-content: space-evenly;
+      grid-row-gap: 20px;
+
+      .grid-item {
+        padding: 20px;
+        margin: auto;
+      }
+    }
+  }
+
+  .philosophy-container {
+    width: 440px;
+    background: $nice-blue;
+    border-radius: 10px 0px 0px 10px;
+    
+    .content {
+      padding: 60px;
+      color: white;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      font-size: $large;
+
+      h2 {
+        text-align: center;
+      }
+    }
+  }
+}
+
+p{
+  max-width: 400px;
+}
+.logo-img {
+  max-width: 400px;
+  display: block;
+}
+
+@media screen and (max-width: 1300px) {
+  .logo-img {
+    max-width: 300px !important;
```

Comment:
Why so many !important?

Reply:
It wouldn't update the attribute unless !important is included. It didn't need to be included for undeclared attributes

Reply:
Gotcha. Okay, just curious!